packages/new-editor/src/components/editor/variables/detail/Variable.tsx

```diff
@@ -0,0 +1,39 @@
+import { Fieldset, Input, SimpleSelect, Textarea, ToolbarTitle } from '@axonivy/ui-components';
+import { Metadata, type Variable } from '../../data/Variable';
+
+type VariableProps = {
+  variable: Variable;
+};
+
+export const VariableDetail = ({ variable }: VariableProps) => {
+  const metadataOptions = Object.keys(Metadata)
+    .filter(key => isNaN(Number(key)))
+    .map(key => {
+      const option = key.toString();
+      return { value: option, label: option };
+    });
+  metadataOptions.unshift({ value: 'None', label: 'None' });
+  const selectedMetadataOption = variable.metadata === undefined ? 'None' : Metadata[variable.metadata];
+
+  return (
+    <>
+      <ToolbarTitle>Variable Configuration</ToolbarTitle>
```

Comment:
why not sidebar header?

Reply:
This is the title underneath the header, not the header itself:
![image](https://github.com/axonivy/config-editor-client/assets/141232142/9409d96f-b842-4719-b84f-ce207523bebd)

Although, maybe I should not use the `ToolbarTitle` component for this?